.coderabbit.yaml

```
@@ -0,0 +1,14 @@
+language: "en-US"
+early_access: false
+reviews:
+  profile: "chill"
+  path_instructions:
+    - path: "**"
+      instructions: |
+        Pull requests that have "TECH_DEBT" in the title should only contain changes related to typos, unused code, linter/IDE suggestions, swagger specification updates, 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)